1st poem

```diff
@@ -0,0 +1,4 @@
+Roses are red,
+Violets are blue,
+This poem is random
```

Comment:
Can you add a comma at the end of this line for consistency?

Reply:
> Can you add a comma at the end of this line for consistency?

Sure!

Reply:
Thanks! Respond to this comment once you've made the commit, please!

Reply:
Ah! I was too slow! Looks great!